packages/builder/src/extrinsic/pallets/xTokens/xTokens.ts

```diff
@@ -173,11 +161,7 @@ export function xTokens() {
                     {
                       Parachain: destination.parachainId,
                     },
-                    {
-                      AccountKey20: {
-                        key: address,
```

Comment:
Adding `Any` here is breaking the integrations using `transferMulticurrencies`, but after taking a look it seems we don't need the `Any` keyword. I'm preparing a fix